mediachain/indexer/mc_ingest.py

```diff
@@ -268,8 +269,8 @@ def cache_image(_id,
         ## TODO: either md5 of native_id or not:
         _id = hashlib.md5(_id).hexdigest()
```

Comment:
I know this isn't new but I prefer to never use md5, even if it's probably OK. sha1 should do nicely at minimal overhead cost